modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/snapshot/IgniteSnapshotManager.java

```diff
@@ -1048,23 +1000,21 @@ public RemoteSnapshotFuture(UUID rmtNodeId, String snpName, int cnt) {
         }
     }
 
-    /**
-     *
-     */
-    private static class SerialExecutor implements Executor {
-        /** */
+    /** */
+    private static class SingleThreadExecutor implements Executor {
```

Comment:
I think it's not a good name now, since tasks executed not in single thread, but can be executed on different threads sequentially. It's better to state in the class name that:
1. This class is wrapper (doesn't implement any functionality to start threads etc, but use underlying executor instead).
2.  Task executed sequentially, but not in the single thread.
So, good names IMO would be something like `SerialExecutorWrapper`, `SequentialExecutorWrapper`, etc